pinconfigs/README.md

```diff
@@ -120,6 +120,10 @@ The AFIO model supports the following fields in the configuration file:
 
   Available configuration fields are:
 
+  - `exclude-pincodes`: A list of pincodes where the signal is not
```

Comment:
mention this is `(optional)`

Reply:
The upper `signal-configs` is optional. Should I add optional to `exclude-pincodes` and `exclude-memories`?

Reply:
yes please

Reply:
done.